diablo/externals/kaltura.py

```diff
@@ -193,7 +193,7 @@ def schedule_recording(
             term_id=term_id,
         )
         recurrences_filter = KalturaScheduleEventFilter(parentIdEqual=kaltura_schedule.id)
-        for recurrence in self._get_events(kaltura_event_filter=recurrences_filter):
+        for recurrence in self._get_events(kaltura_event_filter=recurrences_filter, organize_results=False):
```

Comment:
We're querying for recurrences of the Kaltura schedule and then cancelling instances that have blackout-date conflicts. In this case, we want the raw list of event_json objects.